src/decide.js

```diff
@@ -56,4 +55,13 @@ export class Decide {
             this.instance['compression'] = {}
         }
     }
+
+    parseDecideError(error) {
+        if (!document?.body) {
+            console.log('document not ready yet, trying again in 500 milliseconds...')
```

Comment:
Do we actually want a console log? Can't remember if this gets pulled out when compiled or not?